### DISCUSSION TITLE
Unwanted Duplicate Connections on Mac OS Big Sur - Audio Cutting Out

### DISCUSSION THREAD

**Discussion by bsepanji** [jamulussoftware/jamulus#1917] [Technical Support & Help] [answered]
Hello,

I'm helping a few of my students setup Jamulus on their computers to have rehearsals each week together. One of the students is on a 2020 Macbook running Big Sur, with Jamulus 3.80. Jamulus runs fine for the first 10 - 15 seconds once he connects to a server but then suddenly he will get a duplicate connection of himself show up on the main control panel window. At that point he hears distortion for a good 30 seconds, at which point the duplicate user disappears and the problem is temporarily solved. This happens continuously throughout the session so it has made for a very unpleasant experience so far.

Does anyone have any idea what is going on? He is using the internal microphone of his laptop along with the headphone output (wired). He is also using an ethernet connection.

Many thanks,

Bijan

**Comment by ann0see** [comment, marked as answer]
That’s strange. Usually, if two people from the same client appear that’s due to internet connectivity problems. Does this happen on different servers? Is the connection blocked by a firewall/anti-virus software? Did you restart routers/switches/pc‘s? Did you try different ethernet cables? My 20m cable was broken and produced comparable issues.

**Reply by bsepanji** [reply]
Thank you so much for the response. As you had suggested, the problem was being caused by anti-virus software on the Mac. The issue was resolved by disabling it!

**Reply by ann0see** [reply]
Great to hear your Problem was solved!

Maybe you can even configure your AV software somehow to not manipulate Jamulus traffic.

**Comment by hoffie** [comment]
We've had this happen mainly with people who were switching between wireless and ethernet connectivity (sometimes inadvertently). To a Jamulus server this would look like new connections each time. So maybe check cabling and monitor local ethernet status?

**Reply by bsepanji** [reply]
I checked that too, and it still happened. As explained above, the anti-virus software was the culprit!

Many thanks.
